src/components/TokenBalance.tsx

```diff
@@ -285,11 +289,13 @@ export function HomeTokenBalance() {
               ? styles.totalBalance
               : styles.balance
           }
-          hideBalance={hideBalance}
+          hideBalance={showHideHomeBalancesToggle ? hideBalance : false}
```

Comment:
ok this was clever, I'm not sure I would've thought of that. If we turn off the hide home balances toggle, we need to avoid getting some users into a state where they can't un-hide their balances. Nice.

Reply:
```suggestion
          hideBalance={showHideHomeBalancesToggle && hideBalance}
```
code golf